src/app/portlets/content-types/fields/content-type-fields-drop-zone/content-type-fields-drop-zone.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="content-type-fields-drop-zone__container"
+     dragula="fields-row-bag"
+     [dragulaModel]="fieldRows"
+     data-drag-type="target">
+
+    <content-type-fields-row  *ngFor="let fieldRow of fieldRows"
```

Comment:
So inside the ```.ts``` of this component we only use the property "columns" from the fieldRow attr, why just don't pass column as an attribute? and also make more sense semantically.